extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlExec.java

```diff
@@ -24,11 +24,10 @@
 import java.io.OutputStream;
 
 public class KubectlExec extends Kubectl.ResourceAndContainerBuilder<V1Pod, KubectlExec>
-    implements Kubectl.Executable<V1Pod> {
+    implements Kubectl.Executable<Integer> {
```

Comment:
use Integer as return value type, so `exitCode()` can be removed --- remote process exit code